OsmAnd/src/net/osmand/plus/myplaces/AvailableGPXFragment.java

```diff
@@ -826,6 +869,12 @@ public void renamedTo(File file) {
 		reloadTracks();
 	}
 
+	@Override
+	public void onGpxUpload(String output) {
```

Comment:
I did not find where it can be used, because in SendGpxBottomSheetFragment you send another listener OsmEditingPlugin